virtool/hmm/api.py

```diff
@@ -124,7 +124,7 @@ async def get(self) -> r200[Response]:
 
         return json_response(updates)
 
-    @policy(PermissionsRoutePolicy("app", "virtool", AppPermission.modify_hmm))
+    @policy(PermissionRoutePolicy("space", 0, SpacePermission.MODIFY_HMM))
```

Comment:
```suggestion
    @policy(PermissionRoutePolicy(ResourceType.SPACE, 0, SpacePermission.MODIFY_HMM))
```
Change all instances of these to use ResourceType.